docs/pages/access-controls/sso/oidc.mdx

```diff
@@ -224,6 +224,21 @@ spec:
 A list of available optional prompt parameters are available from the
 [OpenID website](https://openid.net/specs/openid-connect-core-1\_0.html#AuthRequest).
 
+### Optional: Disable Email Verification
+
+By default, Teleport will validate the `email_verified` claim, and users who attempt to sign in without a verified email address will be prevented from doing so.
```

Comment:
```suggestion
By default, Teleport will validate the `email_verified` claim and reject login attempts from users without a verified email address.
```

This is more succinct but it's up to you to decide if it actually sounds better.